src/test/java/uk/gov/ons/ctp/response/sample/Unirest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.ons.ctp.response.sample;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import java.io.IOException;
+
+public class Unirest {
```

Comment:
Maybe rename this to something else then we can import com.mashape.unirest.http.Unirest